client/packages/core/src/authAPI.ts

```diff
@@ -58,6 +68,20 @@ export async function verifyRefreshToken({
   return res;
 }
 
+export async function signInAsGuest({
+  apiURI,
+  appId,
+}: SharedInput): Promise<VerifyResponse> {
+  const res = await jsonFetch(`${apiURI}/runtime/auth/sign_in_guest`, {
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: JSON.stringify({
+      'app-id': appId,
+    }),
+  });
+  return res;
```

Comment:
The `VerifyResponse` type contains a `user` and the`user` has a non-null `email` field, which will be undefined here.

Reply:
Right! Updated the type